krikri.gemspec

```diff
@@ -27,6 +27,7 @@ Gem::Specification.new do |s|
   s.add_dependency "jsonpath"
   s.add_dependency "devise", "~>3.4.1"
   s.add_dependency "resque", "~>1.0"
+  s.add_dependency "dpla-couchrest", "1.2.1.pre.dpla"
```

Comment:
What's the official policy/strategy for `dpla-couchrest`?  Do we want to cut a (non-`pre`) release and/or version it in some way that we can update, in case we need to make changes?